vibes/vibed-ui/src/compositor/Compositor.cpp

```
@@ -0,0 +1,35 @@
+// MIT License
+// Copyright (c) 2026 Palaash
+
+#include "Compositor.h"
+
+namespace compositor {
+
+Compositor::Compositor() {}
+
+Compositor::~Compositor() {}
+
+void Compositor::run() {
```

Cppcheck (2.20.0)

[style] The function 'run' is never used.

(unusedFunction)